src/modules/Microsoft.FinOpsHubs/datasets/deploy.bicep

```diff
@@ -0,0 +1,91 @@
+// Source: 
```

Comment:
Can we move these folders outside of `Microsoft.FinOpsHubs`? I'd rather avoid a fake RP folder that could be easily confused with a real RP.